artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/LargeMessageController.java

```diff
@@ -18,10 +18,9 @@
 
 import java.io.OutputStream;
 
-import org.apache.activemq.artemis.api.core.ActiveMQBuffer;
 import org.apache.activemq.artemis.api.core.ActiveMQException;
 
-public interface LargeMessageController extends ActiveMQBuffer {
```

Comment:
this is an API change, isn't?

it's part of the Core Client API when dealing with large messages...

it's a client API.. I don't think. you can guarantee there are no users using this API.

Reply:
answered on https://github.com/apache/activemq-artemis/pull/3334#discussion_r522248647